public/app/config/collections/memorials/searchTypes.js

```diff
@@ -0,0 +1,23 @@
+(function () {
+    'use strict';
+
+    define(function () {
+        return [
+            {
+                key: '_fulltext',
+                labelText: 'Keyword',
+                glyphicon: 'search'
+            },
+            {
+                key: 'ca_objects.preferred_labels',
+                labelText: 'Item Name',
+                glyphicon: 'certificate'
+            },
+            {
+                key: 'ca_occurrences',
```

Comment:
This hasn't changed, I've just moved the file.  If it's wrong I'll make a separate PR to change it.